internal/job/executor.go

```diff
@@ -180,30 +181,18 @@ func (e *Executor) Run(ctx context.Context) (exitCode int) {
 		return shell.GetExitCode(err)
 	}
 
-	includePhase := func(phase string) bool {
-		if len(e.Phases) == 0 {
-			return true
-		}
-		for _, include := range e.Phases {
-			if include == phase {
-				return true
-			}
-		}
-		return false
-	}
```

Comment:
i have looked at this code and thought "why is this an inline closure? i should make it a method" and then... not for the entire time i've worked here. thank you for making it happen!!!